cmd/kube-proxy/app/server.go

```diff
@@ -146,6 +146,7 @@ func AddFlags(options *Options, fs *pflag.FlagSet) {
 		&options.config.Conntrack.TCPCloseWaitTimeout.Duration, "conntrack-tcp-timeout-close-wait",
 		options.config.Conntrack.TCPCloseWaitTimeout.Duration,
 		"NAT timeout for TCP connections in the CLOSE_WAIT state")
+	fs.BoolVar(&options.config.EnableProfiling, "profiling", options.config.EnableProfiling, "If true enables profiling via web interface on /debug/pprof handler.")
```

Comment:
I thnk it's appropriate to have a flag like this - I am assuming it is not generally going to be on, and it's not a "knob" but a very fundamental operating mode, not something users would change except to debug?